tests/interlocked_win32_ut/mock_interlocked.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "windows.h"
+#include "umock_c/umock_c_prod.h"
+
+MOCKABLE_FUNCTION(, LONG, mock_InterlockedAdd, volatile LONG*, addend, LONG, value);
```

Comment:
>MOCKABLE_FUNCTION [](start = 0, length = 17)

missing __cplusplus thingie, extern "C"{...

Reply:
how did this even compile for the .dll tests... 

---
In reply to: [430698572](https://github.com/Azure/azure-c-util/pull/43#discussion_r430698572) [](ancestors = 430698572)